config/routes.rb

```diff
@@ -115,6 +115,11 @@
 
     namespace :admin do
 
+      get  "/paypal_preferences"                      => "paypal_preferences#index"
+      post "/paypal_preferences/preferences_update"   => "paypal_preferences#preferences_update"
```

Comment:
Rails protip: you can use `resource` instead of `resources` if the resource is a singleton. Using `resource :paypal_preferences, only: [:update, :show]` would create "resourceful" routes for both get and update without `:id` part. Some refactoring would be needed to use `PUT` instead of `POST` for the update action.

http://stackoverflow.com/questions/9194767/difference-between-resource-and-resources-methods